src/PublicApi/WebFileSystem.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.eShopWeb.PublicApi.CatalogItemEndpoints;
+using Newtonsoft.Json;
+
+namespace Microsoft.eShopWeb.PublicApi
+{
+    public class WebFileSystem: IFileSystem
```

Comment:
Ideally the implementation belongs in the Infrastructure project and shouldn't be referenced directly from the front end project.

Reply:
PublicApi it is not in the frontend it is backend. so we need to move the WebFileSystem and IFileSystem to Infrastructure?

Reply:
IFileSystem to Infrastructure done